src/tracing/index.ts

```diff
@@ -18,6 +19,13 @@ window.GIT_COMMIT_HASH = process.env.REACT_APP_GIT_COMMIT_HASH
 const AMPLITUDE_DUMMY_KEY = '00000000000000000000000000000000'
 export const STATSIG_DUMMY_KEY = 'client-0000000000000000000000000000000000000000000'
 
+const beforeSend: Required<ClientOptions>['beforeSend'] = (event: ErrorEvent, hint: EventHint) => {
+  if (event.request?.url) {
```

Comment:
Add a comment above the conditional explaining what you're doing (possibly linking to the referenced blog post?).